pkg/api/types.go

```diff
@@ -89,6 +89,7 @@ type StrategyParameters struct {
 	LabelSelector                     *metav1.LabelSelector
 	NodeFit                           bool
 	IncludePreferNoSchedule           bool
+	ExcludedTaints                    []string
```

Comment:
`pkg/apis/v1alpha1/types.go` needs to be updated as well since we [convert v1alpha1 into the internal](https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/policyconfig.go#L50) + running the deepcopy/conversion generators

Reply:
I've updated v1alpha1/types.go, but `make gen` doesn't seem to be modifying the auto-generated files. I'll look into this.

Reply:
Pushed the auto-generated changes - the issue was https://github.com/kubernetes-sigs/descheduler/pull/316.